Makefile

```diff
@@ -32,6 +32,15 @@
 # make delete-gke-cluster
 #
 ## http://makefiletutorial.com/
+define n
```

Comment:
What's this for?

Reply:
It is used to define a newline in the Makefile to improve error message readability. I look around but could not find a second way to do it.

Reply:
I think the diff is off here. Bad merge?